### ISSUE TITLE
About type chinese

### ISSUE THREAD

**Issue by qux-bbb** [pbek/QOwnNotes#715]
#### I cannot type chinese in the md. My system is kali rolling based on debian 8.

**Comment by pbek** [comment]
Maybe some Chinese users can help you, @LiangWei88 or try to contact some translators on <https://crowdin.com/project/qownnotes/zh-CN/activity>.

**Comment by CJSoldier** [comment]
I have the same problem.
I succeed to compiled qownnotes,but I cannot type chinese.
It's QT creator's problem. I installed QT Creator5.9 on ubunt16.04, but I couldn't type chinese in the QT IDE. QT Creator has some trouble with ficitx, I tried to uninstall my sougou input method and ficitx, Then I tried again to type chinese in QT IDE, I failed.

I'm not familiar with QT, I heard that we should install a plugin for your QT Creator to support Chinese , I will try it later.

**Comment by pbek** [comment]
Thank you for the feedback!

**Comment by CJSoldier** [comment]
@qux-bbb Hey buddy,  I found a solution [here](http://blog.leanote.com/post/ghost54@163.com/%EF%BC%B1t5.9.1%E6%97%A0%E6%B3%95%E8%BE%93%E5%85%A5%E4%B8%AD%E6%96%87%E7%9A%84%E9%97%AE%E9%A2%98%E3%80%80). I can type Chinese in my qownnotes now, cheers. Hope that could help you too.

**Comment by pbek** [comment]
Great news, @CJSoldier. Thank you for finding out!

**Comment by CJSoldier** [comment]
Hi @pbek, I fall in love with qownnotes now. Since the QT does not support css well, now I understand why you didn't give blockquote style. I tried a lot but I can only add a backgroud color to blockquote.
Hope that you could add some style some day.

**Comment by qux-bbb** [comment]
Today, I found that I can type Chinese in the md， but I do noting. Maybe because of the updating.

**Comment by pbek** [comment]
>  I tried a lot but I can only add a backgroud color to blockquote. Hope that you could add some style some day.

can you please be more specific :) I'm not sure I already understand what you are talking about

**Comment by CJSoldier** [comment]
Ah, What I really what to do is to beautify the preview of > quoted text. I tried to add some styles to blockquote tag in toMarkdownHtml method according to your answer [styling preview](https://github.com/pbek/QOwnNotes/issues/164). For example, I added border-left-width:10px, but it seems that QT does not support the syntax. Finally I changed quoted text background color which make me feel that it's quoted text rather than indented text.
